pkg/operator/yugabytedb/controller_test.go

```diff
@@ -1193,7 +1193,10 @@ func simulatePodsRunning(clientset *fake.Clientset, namespace string, podCount i
 			},
 			Status: v1.PodStatus{Phase: v1.PodRunning},
 		}
-		clientset.CoreV1().Pods(namespace).Create(pod)
+		_, err := clientset.CoreV1().Pods(namespace).Create(pod)
+		if err != nil {
```

Comment:
Shouldn't we return an error? Otherwise, the test using this function will be compromised.